cli/cfncluster/cfncluster.py

```diff
@@ -51,9 +47,12 @@ def create(args):
         i = [p[0] for p in config.parameters].index('MasterSubnetId')
         master_subnet_id = config.parameters[i][1]
         try:
-            vpcconn = boto.vpc.connect_to_region(config.region,aws_access_key_id=config.aws_access_key_id,
-                                                 aws_secret_access_key=config.aws_secret_access_key)
-            availability_zone = str(vpcconn.get_all_subnets(subnet_ids=master_subnet_id)[0].availability_zone)
+            ec2 = boto3.client('ec2', region_name=config.region,
+                                 aws_access_key_id=config.aws_access_key_id,
+                                 aws_secret_access_key=config.aws_secret_access_key)
+            availability_zone = ec2.describe_subnets(SubnetIds=[master_subnet_id])\
+                .get('Subnets')[0]\
+                .get('AvailabilityZone')
         except Exception as e:
             logger.critical(e.message)
```

Comment:
there is no "message" in e.
Should be
logger.critical(e.response.get('Error').get('Message'))
?